modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -253,7 +253,7 @@
 :is( .elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-n-tab-title,
 	.elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content .e-n-tab-title ) {
 
-	&.e-normal:hover {
+	&:hover {
```

Comment:
Have you tried this? 

With doing this, we are targetting the `e-normal` directly, without changing the specifity. 

And we are remove the hover effect from the tabs `.e-n-tabs-content`.

```.elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-normal:hover {
		background-color: var( --n-tabs-title-background-color-hover );
		background-image: initial;
	}
```

Reply:
I tried that snippet, it makes no change.
We can take it offline and investigate it together.
Removing the `.e-normal` class decreases the specificity and allows the `:not(:hover)` rule to be assigned to the active tab.